src/core/config/border_agent.h

```diff
@@ -89,6 +89,38 @@
     OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_AGENT_MESHCOP_SERVICE_ENABLE
+ *
+ * Define to 1 to enable Border Agent to manage registering/updating of the mDNS MeshCoP service(s) on the
+ * infrastructure link
+ *
+ * This includes the ephemeral key service when the `OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE` is enabled.
+ */
+#ifndef OPENTHREAD_CONFIG_BORDER_AGENT_MESHCOP_SERVICE_ENABLE
+#define OPENTHREAD_CONFIG_BORDER_AGENT_MESHCOP_SERVICE_ENABLE \
+    (OPENTHREAD_CONFIG_PLATFORM_DNSSD_ENABLE || OPENTHREAD_CONFIG_MULTICAST_DNS_ENABLE)
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_AGENT_MESHCOP_SERVICE_BASE_NAME
+ *
+ * Specifies the base name to construct the service instance name used when advertising the mDNS `_meshcop._udp`
+ * service by the Border Agent.
+ *
+ * Applicable when the `OPENTHREAD_CONFIG_BORDER_AGENT_MESHCOP_SERVICE_ENABLE` feature is enabled.
+ *
+ * The name can also be configured using the `otBorderAgentSetMeshCoPServiceBaseName()` API at run-time.
+ *
+ * Per Thread specification, the service instance should be a user-friendly name identifying the device model or
```

Comment:
```suggestion
 * Per the Thread specification, the service instance should be a user-friendly name identifying the device model or
```

Reply:
Thanks. All resolved in the new pushed commit.